src/gui_impl/dialogs/event_types_selection_dialog.cpp

```diff
@@ -48,8 +48,8 @@ void EventTypesSelectionDialog::buildTree (bool only_existing_events)
     ui_.tree_widget_->setColumnWidth(ID_COLUMN_INDEX_, 0);
 
     ui_.tree_widget_->header()->setSectionResizeMode (QHeaderView::Interactive);
-    ui_.tree_widget_->header()->resizeSection (NAME_COLUMN_INDEX_, 380);
-    ui_.tree_widget_->header()->resizeSection (COLOR_COLUMN_INDEX_, 120);
+    ui_.tree_widget_->header()->resizeSection (NAME_COLUMN_INDEX_, width() / 1.65);
+    ui_.tree_widget_->header()->resizeSection (COLOR_COLUMN_INDEX_, width() / 5.5);
```

Comment:
I like it! And who knows, maybe we will add some additional info the the channels (e.g. labels) in the future so we need the extra space.